misk/src/test/kotlin/misk/ActionsTest.kt

```diff
@@ -1,36 +1,48 @@
 package misk
 
+import misk.web.RequestBody
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
 import kotlin.reflect.full.createType
 import kotlin.test.assertFailsWith
 
 internal class ActionsTest {
-  @Test
-  fun methodAsAction() {
+  @Test fun methodAsAction() {
     val action = TestAction::myActionMethod.asAction()
     assertThat(action.name).isEqualTo("TestAction")
-    assertThat(action.parameterTypes).hasSize(1)
+    assertThat(action.parameterTypes).hasSize(2)
     assertThat(action.parameterTypes[0]).isEqualTo(Int::class.createType())
+    assertThat(action.parameterTypes[1]).isEqualTo(String::class.createType())
+    assertThat(action.requestType).isEqualTo(String::class.createType())
     assertThat(action.returnType).isEqualTo(String::class.createType())
   }
 
-  @Test
-  fun methodReferenceNotAllowedAsAction() {
+  @Test fun noRequestBodyAction() {
+    val action = NoRequestBodyAction::myActionMethod.asAction()
+    assertThat(action.name).isEqualTo("NoRequestBodyAction")
+    assertThat(action.parameterTypes).hasSize(1)
+    assertThat(action.parameterTypes[0]).isEqualTo(Int::class.createType())
+    assertThat(action.requestType).isNull()
+  }
+
+  @Test fun methodReferenceNotAllowedAsAction() {
```

Comment:
Could you add a single sentence what this is testing?

Reply:
these were refactored so the annotation is on the same line as the test. I'm more a fan of the test method name explaining what it's doing. I think the test name is explanatory enough?